server/app/views/BaseHtmlView.java

```diff
@@ -74,8 +74,9 @@ protected static ContainerTag asRedirectButton(ContainerTag buttonEl, String red
   protected static ContainerTag makeSvgTextButton(String buttonText, String svgPath) {
     return TagCreator.button()
         .with(
-            Icons.svg(svgPath, 18).withClasses(Styles.ML_1, Styles.MR_2, Styles.INLINE_BLOCK),
-            span(buttonText));
+            Icons.svg(svgPath, 18)
```

Comment:
FYI, this PR is built on top of #2799, which will be submitted prior to merging this.

Reply:
That PR has been merged and this comment is no longer relevant.